components/profile_popover/profile_popover.jsx

```diff
@@ -345,10 +372,23 @@ class ProfilePopover extends React.Component {
             />
         );
 
+        const supportsChannelAdmin = this.props.isLicensed;
+        const isChannelAdmin = supportsChannelAdmin && (Utils.isChannelAdmin(this.props.isLicensed, this.props.channelMember.roles) || this.props.channelMember.scheme_admin);
+
```

Comment:
I think this check should just be `this.props.channelMember.scheme_admin`. The concept of `channel_admin` as a stand-alone role no longer exists since the new permissions system landed.

Reply:
Also, I'm not convinced about doing a license check here. Channel Admin's *do* exist in TE, it's just their powers are not configurable.

Reply:
Makes sense, I've taken the example from https://github.com/mattermost/mattermost-webapp/blob/master/components/channel_members_dropdown/channel_members_dropdown.jsx#L78, should we adapt it there as well as part of a separate PR?

Reply:
Maybe leave it there for now. It's there to keep backwards compatability while the advanced permissions migration is running. We can delete it in future, but probably not yet. However, it's not needed here as this is a completely new feature, which can remain invisible until the migration completes if someone upgrades from 5.1 directly to 5.6 without any inbetween versions.